api/routes/graphs.py

```
@@ -217,70 +216,47 @@
 
 @graphs_router.post("")
 @token_required
-async def load_graph(request: Request, data: GraphData = None, file: UploadFile = File(None)):
+async def load_graph(request: Request, data: GraphData = None, file: UploadFile = File(None)): # pylint: disable=unused-argument
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)